include/wx/charts/wxchartgrid.h

```diff
@@ -128,6 +128,9 @@ class wxChartGrid : public wxChartElement
     void Shift(double dx,double dy);
     void ChangeCorners(wxDouble minX,wxDouble maxX,
```

Comment:
Let's rename these functions to UpdateAxisLimits to be consistent with the variable names.

Reply:
Also as I mentioned in the comment, I named each axis. So let's not update both axes at the same time but instead have:
void UpdateAxisLimit(const std::string& axisId, wxDouble minX,wxDouble maxX)